backend/src/main/java/hu/flowacademy/worksheet/service/UserService.java

```diff
@@ -49,4 +50,11 @@ private void validateUser(User user) throws ValidationException {
     public List<User> findUserByNameAndEmail(String searchPart) {
         return userRepository.findByEmailContainingOrFirstNameContainingOrLastNameContaining(searchPart, searchPart, searchPart);
     }
+
+    public User setUserActivity(Long id, String status) throws ValidationException {
+        User toChange = userRepository.findById(id).orElseThrow(()-> new ValidationException("No user with provided ID"));
+        toChange.setEnabled(status.toLowerCase().equals(Status.active.name()));
```

Comment:
Ha enumként kapod meg a Status-t, akkor nem is kell equals, elég a hagyományos összehasonlítás.